app/templates/preprints/provider/preprint-detail.hbs

```diff
@@ -161,4 +161,14 @@
         </div>
     </div>
 
+    {{#bs-modal open=showWarning onHide=(action 'hideWarning') as |modal|}}
```

Comment:
This could be `onHide=(action (mut showWarning false))`, which would let you delete `hideWarning`.